modules/openxr/openxr_api.cpp

```diff
@@ -45,10 +45,27 @@
 #include "extensions/openxr_android_extension.h"
 #endif
 
+// We need to have all the platform defines before the Vulkan or OpenGL extensions
+// are included, otherwise we'll only get one graphics API or the other.
+#ifdef VULKAN_ENABLED
+#define XR_USE_GRAPHICS_API_VULKAN
+#endif
+#ifdef GLES3_ENABLED
+#ifdef ANDROID
```

Comment:
Indeed, the `ANDROID` define is now `ANDROID_ENABLED` so change this and in the other places.